projects/github.com/nishanths/license/package.yml

```diff
@@ -0,0 +1,30 @@
+distributable:
+  url: https://github.com/nishanths/license/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+warnings:
+  - vendored
```

Comment:
this is incorrect, but I'll remove it post merge. This warning is intended for “tea didn’t build this. BEWARE!”